source/features/pr-jump-to-first-non-viewed-file.tsx

```diff
@@ -6,8 +6,8 @@ import * as pageDetect from 'github-url-detection';
 import features from '../feature-manager.js';
 
 function jumpToFirstNonViewed(): void {
-	const firstNonViewedFile = $('.file:not([data-file-user-viewed])')!;
-	if (firstNonViewedFile) {
+	const firstNonViewedFile = $('.file[data-details-container-group="file"]:not([data-file-user-viewed])')!;
```

Comment:
Adding this selector prevents selecting the wrong DOM element (maybe the classes in changed in the past)